english/BayesTheorem/puppies/bayes-and-puppies.md

```diff
@@ -0,0 +1 @@
+When learning about [Bayes Theorem](https://en.wikipedia.org/wiki/Bayes'_theorem), why not learn with [puppies](https://sites.google.com/site/doingbayesiandataanalysis/)
```

Comment:
@casey-collab, please take another look.  I left only the link for puppies this time.